ci/scripts/python_wheel_manylinux_build.sh

```diff
@@ -89,31 +89,32 @@ pushd /tmp/arrow-build
 # https://github.com/aws/aws-sdk-cpp/issues/1809 is fixed and vcpkg
 # ships the fix.
 cmake \
-    -DARROW_BROTLI_USE_SHARED=OFF \
     -DARROW_BUILD_SHARED=ON \
     -DARROW_BUILD_STATIC=OFF \
     -DARROW_BUILD_TESTS=OFF \
+    -DARROW_COMPUTE=ON \
+    -DARROW_CSV=ON \
     -DARROW_DATASET=${ARROW_DATASET} \
+    -DARROW_DATASET=ON \
```

Comment:
```suggestion
    -DARROW_DATASET=ON \
```

Reply:
Thanks but I choose `-DARROW_DATASET=${ARROW_DATASET}` for consistency.